### ISSUE TITLE
how does repo-picker flow work?

### ISSUE THREAD

**Issue by ckingbailey** [ckingbailey/publiccode-pusher#2]
# How @ckingbailey thought it should work
1. user authenticates
2. user picks brigade
3. user picks repo
4. fill out for repo

# 🌟@plocket's recommendation:🌟
// This is the one we're going with
1. provide user with free text field where they put the url of the target repo*
2. form targets chosen repo

\* This should affect the OAuth flow:
Only authorize the user after they've entered a URL to target.**

\** BONUS: waiting to authorize the user until they enter a target URL allows us to check that the user has _write access_ to the target repo before we show them the form.
If they don't have write access we'll need to create a fork for them, or show them how.

**Comment by plocket** [comment]
Flow based on [group-determined priorities](https://docs.google.com/spreadsheets/d/1KP07YcZqe2DfszMnuJyfpsC-mp4rLdlZbE6DHA3CJdw/edit#gid=1144329632)

![user_flow](https://user-images.githubusercontent.com/52798256/73872397-a522e380-481d-11ea-9a44-6e830d2bbfa8.jpeg)

**Comment by plocket** [comment]
Alternative to a fork - they could make an issue with the file included in it. Feasibility?

**Comment by ckingbailey** [comment]
Oh that's an interesting idea! How do you see the file being included? As in-place text, like:
```yaml
publiccodeYmlVersion: "0.2"

name: Medusa
url: "https://example.com/italia/medusa.git"
softwareVersion: "dev"    # Optional
releaseDate: "2017-04-15"
platforms:
  - web
etc: etc
```
?

**Comment by plocket** [comment]
Yeah, that's what I was thinking. It's possible to drag a file in here, but only docx, pdf, txt, and some other weird stuff. I don't think that's really useful.

Possibly we could set up a link to a file that's stored somewhere temporarily. Not sure that's quite worth it.

**Comment by ckingbailey** [comment]
resolved by #13
